testsuite/MDAnalysisTests/analysis/test_bat.py

```diff
@@ -127,3 +128,11 @@ def test_bat_incorrect_dims(self, bat_npz):
         errmsg = 'Dimensions of array in loaded file'
         with pytest.raises(ValueError, match=errmsg):
             R = BAT(selected_residues, filename=bat_npz)
+
+    def test_bat_transformation(self, selected_residues, bat):
+        R = BAT(selected_residues)
+        pre_transformation = copy.deepcopy(bat[0])
+        R.Cartesian(bat[0])
+        assert_almost_equal(pre_transformation, bat[0], 5,
+                            err_msg="error: Bat.Cartesian"
+                                    "modified input data")
```

Comment:
```suggestion
                            err_msg="error: BAT.Cartesian modified input data")
```

This is missing a space. If it triggers a PEP8 error, consider using the formatting I suggested previously.